devdoc/crt_abstraction_requirements.md

```diff
@@ -94,6 +92,8 @@ int sprintf_s(char* dst, size_t dstSizeInBytes, const char* format, ... );
 
 **SRS_CRT_ABSTRACTIONS_99_027: [** sprintf_s shall return the number of characters stored in dst upon success.  This number shall not include the terminating null character. **]**
 
+**SRS_CRT_ABSTRACTIONS_02_006: [** If the conversion fails for any reason (for example, insufficient buffer space), a non-zero return value shall be supplied and size_tToString shall fail. **]**
```

Comment:
>size_tToString [](start = 162, length = 14)

`sprintf_s` ?

Reply:
Fixed.

---
In reply to: [362558963](https://github.com/Azure/azure-c-util/pull/7#discussion_r362558963) [](ancestors = 362558963)